tests/icalmonthly.txt

```diff
@@ -0,0 +1,17 @@
+BEGIN:VCALENDAR
+VERSION:2.0
+X-WR-CALNAME:Privat
+X-APPLE-CALENDAR-COLOR:#FF2968
+X-WR-CALDESC:
+BEGIN:VEVENT
+CREATED:20090213T195947Z
+UID:M2CD-1-1-5FB000FB-BBE4-4F3F-9E7E-217F1FF97208
+RRULE:FREQ=MONTHLY;BYMONTHDAY=1;WKST=SU;COUNT=25
+DTSTART;VALUE=DATE:20180701
+DTEND;VALUE=DATE:20180702
+SUMMARY:Monthly
+LAST-MODIFIED:20110429T222101Z
+DTSTAMP:20170630T105724Z
+SEQUENCE:0
+END:VEVENT
+END:VCALENDAR

```

Comment:
Should this be called `icalmonthly.ics`?

Reply:
renamed.